src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,42 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        int initialPositionX = robot.getX();
```

Comment:
good start
but, let's remove variable `initialPositionX` and use robot.getX() or robot.getY() everywhere